centreon/www/widgets/src/centreon-widget-graph/src/LineChart.tsx

```diff
@@ -158,14 +169,20 @@ const WidgetLineChart = ({
     );
   }
 
+  const barChartOrientation = cond([
+    [equals('horizontal'), always('vertical')],
+    [equals('vertical'), always('horizontal')],
+    [T, identity]
+  ])(panelOptions.orientation) as 'auto' | 'horizontal' | 'vertical';
+
   return (

```

Comment:
It might be better to only change the label names in the widget properties to avoid changing the visualization for users who upgrade their platforms